net-analyzer/ostinato/ostinato-1.3.0.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit qmake-utils
+
+DESCRIPTION="Packet generator and analyzer"
+HOMEPAGE="https://ostinato.org/"
+SRC_URI="https://github.com/pstavirs/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
```

Comment:
should be `GPL-3+`, see: https://github.com/pstavirs/ostinato/blob/master/client/main.cpp#L6-L9